### ISSUE TITLE
Markers not rendering until zoom change

### ISSUE THREAD

**Issue by djpetenice** [venits/react-native-map-clustering#75]
I'm setting the map region based on geolocation and once the map renders the markers don't show until a zoom change. Turning clustering to false shows the markers without any issues. All working fine before adding the library.

Any ideas on how to resolve?

**Comment by djpetenice** [comment]
Also I've noticed toggling clustering prop to true is having an impact on animateToRegion - only works intermittently.

**Comment by tomassvatek** [comment]
@djpetenice I have the same problem. Did you find any solution?

**Comment by ViktorVojtek** [comment]
Hey guys!
I experience the same behaviour if I set or change region by animateToRegion method of the original maps component from airbnb/community. I guess it's because the MapView component of this module computes clusters/markers based on the region prop.

As far as you will not update the region prop of MapView component the clusters/markers will not be computed. I guess cluster computation is automatically triggered on onRegionChangeComplete methode for example. So if we pan on map or we change zoom onRegionChangeComplete will trigger and markers/clusters shows up.

I have solved it by updating a region prop in HOC component.
Something like:

setRegion(region) {
    const { ready } = this.state;

    if (ready) {
      setTimeout(() => {
        this.map.root.animateToRegion(region, 2000);
       // It's a pity there is no callback in animateToRegion method :/
        setTimeout(() => this.setState({ region }), 2000);
      }, 10);
    }
  }
